EXAMPLE.md

```diff
@@ -0,0 +1,302 @@
+# Running Tests in an openSUSE VM
+
+This is an example of setting up an [openQA][openqa] server with this set of
+tests and a worker inside an openSUSE QEMU VM.
```

Comment:
The setup described here (running openqa in a VM) is unlikely to be enough to run all the tests, especially some integration tests are rather heavy. So, please add a note what's the intended use and for which kind of tests it's enough.